lib/src/utils/uri_extension.dart

```diff
@@ -59,19 +62,22 @@ extension MxcUriExtension on Uri {
   ///
   /// Important! To use this link you have to set a http header like this:
   /// `headers: {"authorization": "Bearer ${client.accessToken}"}`
-  Future<Uri> getThumbnailUri(Client client,
-      {num? width,
-      num? height,
-      ThumbnailMethod? method = ThumbnailMethod.crop,
-      bool? animated = false}) async {
+  Uri getThumbnailUri(
```

Comment:
I think we should deprecate the functions to get a raw URL as an SDK user. They might be useful internally, but SDK users will need to manually set auth headers when using any of these URIs. This means you basically want to use the thumbnail and download functions from the SDK anyway, never really the *Uri functions, I think.

Reply:
Okay, so some clients are using this to feed Image.network and similar, so those would benefit from keeping these around. However I think we can at least deprecate the extensions on Uri and only expose 2 functions returning a Uri instead if 4+.